baklava/package.json

```
@@ -1,78 +1,81 @@
 {
-  "name": "dogehouse",
-  "version": "1.0.0",
-  "description": "",
-  "main": "./dist/electron.js",
-  "scripts": {
-    "compile": "tsc",
-    "watch": "tsc -w",
-    "dev": "npm run compile && electron ./dist/electron.js",
-    "build": "npm run compile && electron-builder -mwl --win msi",
-    "build:mac": "npm run compile && electron-builder --mac",
-    "build:win": "npm run compile && electron-builder --win",
-    "build:linux": "npm run compile && electron-builder --linux"
-  },
-  "keywords": [],
-  "author": "Ben Awad <benawadapps@gmail.com>",
-  "homepage": "https://github.com/benawad/dogehouse/",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/benawad/dogehouse"
-  },
-  "bugs": {
-    "url": "https://github.com/benawad/dogehouse/issues"
-  },
-  "license": "MIT",
-  "devDependencies": {
-    "@types/node": "^14.14.31",
-    "electron": "^11.3.0",
-    "electron-builder": "^22.10.5",
-    "typescript": "^4.2.2"
-  },
-  "build": {
-    "appId": "com.electron.dogehouse",
-    "productName": "DogeHouse",
-    "asar": true,
-    "mac": {
-      "icon": "icons/icon.png",
-      "category": "public.app-category.social-networking",
-      "entitlements": "settings/entitlements.mac.plist",
-      "extendInfo": {
-        "NSMicrophoneUsageDescription": "Mic access for speeach input in voice."
-      },
-      "target": [
-        "dmg"
-      ]
+    "name": "dogehouse",
+    "version": "1.0.0",
+    "description": "",
```

I don't know about you, but Electron build keeps barking at me to define a description 🤣 

Don't have to add one, just a thought. Even something as simple as this would make my cli happy 🤣 
```suggestion
    "description": "Taking voice conversations to the moon 🚀",
```